rebar.config.erl

```diff
@@ -482,10 +482,15 @@ emqx_etc_overlay_per_edition(ee) ->
     ].
 
 get_vsn(Profile) ->
-    %% to make it compatible to Linux and Windows,
-    %% we must use bash to execute the bash file
-    %% because "./" will not be recognized as an internal or external command
-    os_cmd("pkg-vsn.sh " ++ atom_to_list(Profile)).
+    case os:getenv("PKG_VSN") of
+        false ->
+            %% to make it compatible to Linux and Windows,
+            %% we must use bash to execute the bash file
+            %% because "./" will not be recognized as an internal or external command
```

Comment:
this comment should be moved down as the function doc for `os_cmd/1`
ref: 65bb71ccca

Reply:
later changed in this commit:
4e64010a3b
but left the comment behind.